packages/icons/src/Person.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+import { createIcon } from '@manifest-ui/icon';
+
+export const Person = createIcon(
```

Comment:
Not blocking comment: Should we work to get these in the system documentation?

Reply:
Definitely! I'm curious as to how we want to represent these